docs/rebuild_admin.rst

```diff
@@ -0,0 +1,404 @@
+Rebuilding an Admin Workstation USB
+-----------------------------------
+
+In cases where an Admin Workstation stick has been lost or destroyed, and no
```

Comment:
- italicize Admin Workstation
- maybe "USB stick" instead of just "stick"

Reply:
fixed